build-fail-blacklist

```diff
@@ -36,7 +36,8 @@ recipes/emboss
 recipes/embassy-phylip
 recipes/triform2
 recipes/haploclique/1.3.1
-recipes/genomedata
+recipes/segtools
+recipes/transrate-tools
```

Comment:
Adding those two became necessary to get it to not build those?

Reply:
Must have been a mistake I think. Merge conflict said that master had those two lines (segtools and transrate) in it while my version had genomedata removed. I just wanted master without the genomedata line so I kept the lines from master and it seemed to be marked as additions?

Transrate is oddly enough already in the blacklist but segtools, which previously was, is not anymore on master. I will adjust accordingly so this only accidentally readds segtools. I will fix.